src/execution_tree/primitives/dot_operation.cpp

```diff
@@ -193,10 +189,18 @@ namespace phylanx { namespace execution_tree { namespace primitives
                                 "dimensions");
                     }
 
-                    double result =
-                        Eigen::Map<Eigen::VectorXd>(lhs.data(), lhs.size()).dot(
-                            Eigen::Map<Eigen::VectorXd>(rhs.data(), rhs.size()));
+                    // HACK: I don't know what this is so disabled it.
+                    HPX_THROW_EXCEPTION(hpx::not_implemented,
```

Comment:
This is supposed to handle the case where the left hand operand is a 'transposed' vector (i.e. a `Nx1` matrix).